### PR TITLE
Move the ID of the button to the parent element

### PR DESCRIPTION
**Pull request by TwistMeister**
Due to a small UX flaw I moved the ID from the button to the parent/root element of the plugin instance. With the ID on the button in the tiles view the tile went out of the scroll view, with the button aligning with the top of the view. Instead of the whole plugin aligning at the top of the view.

**Before this fix**

https://user-images.githubusercontent.com/16188579/233085911-8f7e6bf3-83bc-4fc5-b665-79b77dea5ed0.mov

**After this fix**

https://user-images.githubusercontent.com/16188579/233085580-1cd84a84-2d40-4441-abd1-ef60e5361a9e.mov

### CODE REVIEW

File: rocky/katalogus/templates/katalogus.html [hunk 1]
```diff
@@ -50,7 +50,7 @@ <h1>{% translate "KAT-alogus" %}</h1>
                             </thead>
                             <tbody>
                                 {% for plugin in object_list %}
-                                    <tr>
+                                    <tr id="enableform_{{ plugin.id|slugify }}">
```
Comment (dekkers) [inline rocky/katalogus/templates/katalogus.html:53]: Should we still name the id enableform if it is put on the row instead of the form?

**Comment by Darwinkel** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make kat`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* Fix works as intended.

**Review by Darwinkel — DISMISSED** [review]

File: rocky/katalogus/templates/katalogus.html (hunk 1 above)
Comment (ammar92) [reply rocky/katalogus/templates/katalogus.html:53]: Indeed, it doesn't make sense to use that prefix anymore. E.g. `plugin_` should suffice, but it also makes sense to not use any prefix

Comment (underdarknl) [reply rocky/katalogus/templates/katalogus.html:53]: agreed, we should call it enableplugin instead.

Comment (TwistMeister) [reply rocky/katalogus/templates/katalogus.html]: Yes, it was a bit lazy to just move the ID and not rethink its prefix. Just fixed it and altered it to "plugin_"

**Review by dekkers — APPROVED** [review]

**Pull request merged**